### PR TITLE
Feature/javascript

### PR DESCRIPTION
**Pull request by tarek102**
Update index.html
Create script.js
Edit styles.css
Add popup functionality
Once clicked on link the window goes to the selected section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,7 +20,13 @@ <h5>9:41</h5>
     </div>
     <div class="lower-nav">
       <h3>Tarek Yosry</h3>
-      <img src="./images/Union.png" alt="hamburger menu">
+      <img class="hamburger-menu" src="./images/Union.png" alt="hamburger menu">
+      <ul class="pop-up">
+        <li><a href="">Portfolio</a></li>
+        <li><a href="">About</a></li>
+        <li><a href="">Contact</a></li>
+      </ul>
```
Comment (thecodechaser) [inline index.html:28]: - [ ] You did a good job on the mobile menu functionalities but there are some issues you need to fix. Please use correct color `#FFF5E1` for mobile menu options.

File: index.html [hunk 2]
```diff
@@ -19,7 +20,13 @@ <h5>9:41</h5>
...
+      <i class="bi bi-x-lg close-icon"></i>
```
Comment (thecodechaser) [inline index.html:29]: - [ ] The close(X) button is a little bigger than available on the template. Please decrease the size of the close(X) button and make it matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A838) design.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @tarek102, and @IamYunusAli👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [ ] In the README file, Please add your partner @IamYunusAli in the author section.
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @tarek102, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions
- [ ] [OPTIONAL] Kindly make when the menu opened I can't scroll to the page for a better user experience.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**